src/ir/operator.h

```diff
@@ -30,22 +27,14 @@ namespace raksha::ir {
 // operations.
 class Operator {
  public:
-  Operator(absl::string_view name, std::unique_ptr<Block> implementation)
-      : name_(name), implementation_(std::move(implementation)) {
-    CHECK(implementation_ != nullptr);
-  }
+  Operator(absl::string_view name) : name_(name) {}
 
   const std::string& name() const { return name_; }
```

Comment:
Should this just return an `absl::string_view`?

Reply:
Ah, I meant to change this use, but looks like I left it out. Will do now.